### ISSUE TITLE
Can't use `launchTargetPath` in `launch.json` until `Select a target to debug`

### ISSUE THREAD

**Issue by pweiskircher** [vector-of-bool/vscode-cmake-tools#461]
### Brief Issue Summary

When trying to use `${command:cmake.launchTargetPath}` in `launch.json` it wouldn't work until I used `Select a target to debug`. 

### Expected:

1. CMake: Configure
1. CMake: Set Default Target
1. Execute `launch.json` target that includes `${command:cmake.launchTargetPath}` as program.
1. The program gets executed.

### Apparent Behavior:

1. CMake: Configure
1. CMake: Set Default Target
1. Execute `launch.json` target that includes `${command:cmake.launchTargetPath}` as program.
1. Error message in `CMake/Build`:

```
[main] =======================================================
[main] No executable target was found to launch. Please check:
[main]  - Have you called add_executable() in your CMake project?
[main]  - Have you executed a successful CMake configure? 
[main] No program will be executed
[extension] [6320] cmake.launchTargetPath finished (returned null)
```

To fix this, one can do a `CMake: Select a target to debug`. @vector-of-bool mentioned in the support chat that this should happen automatically.

### CMake Tools Log

```
-- Configure

[main] Starting new CMake driver
[main] Starting CMake driver
[driver] CMakeDriver Kit set to Clang 9.1.0
[driver] Run _refreshExpansions
[cms-driver] Run doRefreshExpansions
[driver] Run _refreshExpansions cb
[cms-client] Started new CMake Server instance with PID 29402
[cache] Reading CMake cache file /Users/pat/Work/PSPDFKit/core/build-vscode/CMakeCache.txt
[cache] Parsing CMake cache string
[driver] Setting new variant , Emit debug information without performing optimizations
[driver] Run _refreshExpansions
[cms-driver] Run doRefreshExpansions
[driver] Run _refreshExpansions cb
[cms-client] Started new CMake Server instance with PID 29403
[cache] Reading CMake cache file /Users/pat/Work/PSPDFKit/core/build-vscode/CMakeCache.txt
[cache] Parsing CMake cache string
[driver] Start configure  
[driver] Runnnig pre-configure checks and steps
[driver] Using compilers in Clang 9.1.0 for configure
[driver] Run _refreshExpansions
[cms-driver] Run doRefreshExpansions
[driver] Run _refreshExpansions cb
[cmake] Building for system : Darwin
[cmake] CMake version: 3.11.3
[cmake] Version: 4.1.0
[cmake] Build type: Debug
[cmake] Configuring done
[cmake] Generating done
[extension] [2943] cmake.configure finished (returned 0)

-- Set Default Target

[extension] [9256] cmake.setDefaultTarget started
[extension] [9256] cmake.setDefaultTarget finished (returned undefined)

-- Trying to run with launch.json

[extension] [1024] cmake.launchTargetPath started
[main] =======================================================
[main] No executable target was found to launch. Please check:
[main]  - Have you called add_executable() in your CMake project?
[main]  - Have you executed a successful CMake configure? 
[main] No program will be executed
[extension] [1024] cmake.launchTargetPath finished (returned null)

-- Select a target to debug

[extension] [7896] cmake.selectLaunchTarget started
[extension] [7896] cmake.selectLaunchTarget finished (returned "/Users/pat/Work/PSPDFKit/core/build-vscode/PSPDFCore-Cpp-Mac-GTests/pspdfcore_tests")

-- After `Select a Target to Debug`

[build] Starting build
[driver] Start build pspdfcore_tests
[driver] Runnnig pre-configure checks and steps
[proc] Executing command: /usr/local/bin/cmake --build /Users/pat/Work/PSPDFKit/core/build-vscode --config Debug --target pspdfcore_tests -- -j 18
[build] ninja: no work to do.
[driver] Run _refreshExpansions
[cms-driver] Run doRefreshExpansions
[driver] Run _refreshExpansions cb
[build] Build finished with exit code 0
[extension] [7967] cmake.launchTargetPath finished (returned "/Users/pat/Work/PSPDFKit/core/build-vscode/PSPDFCore-Cpp-Mac-GTests/pspdfcore_tests")
```

### Platform and Versions

- **Operating System**: macOS 10.13.5
- **CMake Version**: 3.11.3
- **VSCode Version**: 1.24
- **CMake Tools Extension Version**: 1.0.1

**Comment by vector-of-bool** [comment]
Looks like this was "accidentally" fixed in 15e392e7.

**Comment by vector-of-bool** [comment]
Should be fixed in 1.1.0
